cypress/support/commands.js

```diff
@@ -57,3 +57,15 @@ Cypress.Commands.add("isNotInViewport", (element) => {
     expect(rect.top).to.be.lessThan(0 - rect.height);
   });
 });
+
+Cypress.Commands.add("triggerPasteEvent", ({ destinationSelector, pastePayload, pasteType = "text/plain" }) {
```

Comment:
I like adding this as a command 👍 Do you think we should change its name though to just `paste` to make the API more aligned with Cypress.

Reply:
Agree.